CondFormats/CTPPSReadoutObjects/xml/rpix_mapping_220_far_TEST.xml

```diff
@@ -0,0 +1,59 @@
+          <roc id="5" hw_id="" SubSystemId="RPix" ROCinChannel="1" FEDId="1294" FMC="0" FEDChannel="10" />
+        </rpix_plane>
+        <rpix_plane id="2">
+          <roc id="0" hw_id="" SubSystemId="RPix" ROCinChannel="0" FEDId="1294" FMC="0" FEDChannel="7" />
+          <roc id="1" hw_id="" SubSystemId="RPix" ROCinChannel="1" FEDId="1294" FMC="0" FEDChannel="7" />
+          <roc id="2" hw_id="" SubSystemId="RPix" ROCinChannel="2" FEDId="1294" FMC="0" FEDChannel="7" />
+          <roc id="3" hw_id="" SubSystemId="RPix" ROCinChannel="0" FEDId="1294" FMC="0" FEDChannel="8" />
+          <roc id="4" hw_id="" SubSystemId="RPix" ROCinChannel="1" FEDId="1294" FMC="0" FEDChannel="8" />
+          <roc id="5" hw_id="" SubSystemId="RPix" ROCinChannel="2" FEDId="1294" FMC="0" FEDChannel="8" />
+        </rpix_plane>
+        <rpix_plane id="3">
+          <roc id="0" hw_id="" SubSystemId="RPix" ROCinChannel="0" FEDId="1294" FMC="0" FEDChannel="5" />
+          <roc id="1" hw_id="" SubSystemId="RPix" ROCinChannel="1" FEDId="1294" FMC="0" FEDChannel="5" />
+          <roc id="2" hw_id="" SubSystemId="RPix" ROCinChannel="2" FEDId="1294" FMC="0" FEDChannel="5" />
+          <roc id="3" hw_id="" SubSystemId="RPix" ROCinChannel="0" FEDId="1294" FMC="0" FEDChannel="6" />
+          <roc id="4" hw_id="" SubSystemId="RPix" ROCinChannel="1" FEDId="1294" FMC="0" FEDChannel="6" />
+          <roc id="5" hw_id="" SubSystemId="RPix" ROCinChannel="2" FEDId="1294" FMC="0" FEDChannel="6" />
+        </rpix_plane>
+        <rpix_plane id="4">
+          <roc id="0" hw_id="" SubSystemId="RPix" ROCinChannel="0" FEDId="1294" FMC="0" FEDChannel="3" />
+          <roc id="1" hw_id="" SubSystemId="RPix" ROCinChannel="1" FEDId="1294" FMC="0" FEDChannel="3" />
+          <roc id="2" hw_id="" SubSystemId="RPix" ROCinChannel="2" FEDId="1294" FMC="0" FEDChannel="3" />
+          <roc id="3" hw_id="" SubSystemId="RPix" ROCinChannel="0" FEDId="1294" FMC="0" FEDChannel="4" />
+          <roc id="4" hw_id="" SubSystemId="RPix" ROCinChannel="1" FEDId="1294" FMC="0" FEDChannel="4" />
+          <roc id="5" hw_id="" SubSystemId="RPix" ROCinChannel="2" FEDId="1294" FMC="0" FEDChannel="4" />
+        </rpix_plane>
+        <rpix_plane id="5">
+          <roc id="0" hw_id="" SubSystemId="RPix" ROCinChannel="0" FEDId="1294" FMC="0" FEDChannel="1" />
+          <roc id="1" hw_id="" SubSystemId="RPix" ROCinChannel="1" FEDId="1294" FMC="0" FEDChannel="1" />
+          <roc id="2" hw_id="" SubSystemId="RPix" ROCinChannel="2" FEDId="1294" FMC="0" FEDChannel="1" />
+          <roc id="3" hw_id="" SubSystemId="RPix" ROCinChannel="0" FEDId="1294" FMC="0" FEDChannel="2" />
+          <roc id="4" hw_id="" SubSystemId="RPix" ROCinChannel="1" FEDId="1294" FMC="0" FEDChannel="2" />
+          <roc id="5" hw_id="" SubSystemId="RPix" ROCinChannel="2" FEDId="1294" FMC="0" FEDChannel="2" />
+        </rpix_plane>
+      </rp_detector_set>
+    </station>
+  </arm>
+
+</top>

```

Comment:
@fabferro I mean, someone from the release managers (I guess) has to create a repository called `CondFormats/CTPPSReadoutObjects` [here](https://github.com/cms-data).
You then would need to make a PR there to upload your data there and remove it here.
For the removal it would be best if you go back in history and never add the files in order to avoid them from cluttering the repository at all.
You can do this by interactive rebase (`git rebase -i <commit ID before the addition of the files>`).
Note, that you should expect your files in the end to be in `data`, not `xml`.